niri-ipc/src/lib.rs

```diff
@@ -668,6 +668,8 @@ pub enum LayoutSwitchTarget {
     Next,
     /// The previous configured layout.
     Prev,
+    /// The specific layout by index.
+    ByIndex(u32),
```

Comment:
```suggestion
    Index(u8),
```
Name consistent with WorkspaceReferenceArg, type consistent with KeyboardLayouts::current_idx

Reply:
Done